django/api/services/calculate_rebate.py

```diff
@@ -0,0 +1,70 @@
+from ..models.go_electric_rebate_application import GoElectricRebateApplication
+from ..models.household_member import HouseholdMember
+from ..settings import INCOME_REBATES
+
+
+def calculate_rebate_amount(cra_response, application_id):
+    def check_individual(primary_income):
+        if primary_income is None:
+            return "not approved"
+        primary_income = int(primary_income)
+        if primary_income > INCOME_REBATES.get("C").get("individual_income"):
+            return "not approved"
+        elif primary_income <= INCOME_REBATES.get("A").get("individual_income"):
+            return "A"
+        elif primary_income <= INCOME_REBATES.get("B").get("individual_income"):
+            return "B"
+        elif primary_income <= INCOME_REBATES.get("C").get("individual_income"):
+            return "C"
+
+    def check_household(primary_income, secondary_income):
```

Comment:
not sure if any of these should be inner functions.

Reply:
Not remove necessarily, but why is it an inner function?

Reply:
moved